src/core/algorithms/gwo.cpp

```diff
@@ -0,0 +1,68 @@
+#include <stdexcept>
+#include <core/algorithms/gwo.h>
+
+namespace dnn_opt
+{
+namespace core
+{
+namespace algorithms
+{
+
+void gwo::optimize()
+{
+
+}
+
+solution* gwo::get_best()
+{
+
+}
+
+void gwo::set_params(std::vector<float> &params)
+{
+
+}
+
+void gwo::reset()
+{
+
+}
+
+void gwo::update_elite()
+{
+  float size = get_solutions()->size();
+  for(int i = 0; i < size; i++){
```

Comment:
- Braces are in a single line
- You should leave a single empty line after variable declaration block